<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="solutionStackedWidget">
      <widget class="QWidget" name="chartPage">
       <layout class="QGridLayout" name="chartLayout"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="inputGrid" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Коэффициент понижения температуры</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="tempCoeffSpinBox">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.980000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="minTempSpinBox">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="queenCountSpinBox">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Количество шагов при постоянном значении температуры</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="solveButton">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Решить</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Максимальная температура</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Минимальная температура</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="switchButton">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Показать доску</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Количество ферзей</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="maxTempSpinBox">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
         <property name="value">
          <double>30.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="stepCountSpinBox">
         <property name="font">
          <font>
           <family>Source Code Pro Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
